.gitignore

```diff
@@ -35,6 +35,7 @@ timeoverflow_development
 timeoverflow_test
 
 .env
+config/database.yml
```

Comment:
And that's the reason of my proposal. Having peer ident in a local postgres setup seems like a very reasonable default. And this requires having 'null' as username and password.

As always, more expert people can use anything from the full DATABASE_URL to the specific DB_HOST/DB_USERNAME/DB_PASSWORD env variables.